tests/test_claude_wrapper_hooks.py

```
@@ -508,6 +529,184 @@ def test_live_socket_preserves_only_listed_claude_auth_keys(failures: list[str])
+    # Pins the precedence between the explicit-opt-in key list
+    # (CMUX_PRESERVE_CLAUDE_AUTH_SELECTION_ENV_KEYS) and the Vertex/Bedrock
+    # auto-preserve introduced for #3641 / #3638: the key list adds entries
+    # to preservation, it does NOT exclude keys from auto-preserve.
+    inherited = {
+        "CMUX_PRESERVE_CLAUDE_AUTH_SELECTION_ENV": "1",
+        "CMUX_PRESERVE_CLAUDE_AUTH_SELECTION_ENV_KEYS": "ANTHROPIC_API_KEY",
+        "ANTHROPIC_API_KEY": "explicitly-listed-key-must-survive",
+        "CLAUDE_CODE_USE_VERTEX": "1",
+        "ANTHROPIC_MODEL": "claude-sonnet-4-5@20250929",
+    }
+    code, auth_env, real_argv, stderr = run_wrapper_auth_env(
```

Ruff (0.15.12)

[warning] Unpacked variable `real_argv` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)